File: week04-1.cpp
// week04-1
long long coloredCells(int n) {
    long long ans = 0;
    for (long long i = 1; i < n * 2; i += 2) {
        ans += i;
    }
    for (long long i = 1; i < n * 2 - 1; i += 2) {
        ans += i;
    }
    return ans;
}

File: week04-1.py
# week04-1
# 2579. Count Total Number of Colored Cells
class Solution:
    def coloredCells(self, n: int) -> int:
        ans = 0
        for i in range(1, n * 2, 2):
            ans += i
        for i in range(1, n * 2 - 1, 2):
            ans += i
        return ans
